pkg/policy/policy_test.go

```diff
@@ -631,6 +631,7 @@ func (s *PolicyTestSuite) TestNodeMerge(c *C) {
 	c.Assert(aNode.CanMerge(&unmergeableNode), Not(Equals), nil)
 
 	modified, err = aNode.Merge(&unmergeableNode)
+	c.Assert(modified, NotNil)
```

Comment:
`modified` should be `false` here so this should probably be `c.Assert(modified, Equals, false)`